.github/pull_request_template.md

```diff
@@ -16,25 +20,43 @@
     of them would change the meaning incorrectly. The only changes to be made are to add a description text under the
     description heading and to add a 'x' to the correct box.
 -->
-    [ ] Fix
-    [ ] Refactor
-    [ ] Target update
-    [ ] Functionality change
-    [ ] Docs update
-    [ ] Test update
-    [ ] Breaking change
+    [] Patch update (Bug fix / Target update / Docs update / Test update / Refactor)
+    [] Feature update (New feature / Functionality change / New API)
+    [] Major update (Breaking change E.g. Return code change / API behaviour change)
+
+----------------------------------------------------------------------------------------------------------------
+### Test results (*required*)
 
-### Reviewers
+<!--
+    Optional
+    For example, add test results for new target
+-->
+    [] No Tests required for this change
+    [] These PR changes will be covered by existing tests and CI
+    [] Tests supplied below
+    
+    
+----------------------------------------------------------------------------------------------------------------
+### Reviewers (*optional*)
 
 <!--
     Optional
     Request additional reviewers with @username
 -->
 
-### Release Notes
+----------------------------------------------------------------------------------------------------------------
+### Release Notes (*required for feature/major PRs*)
 
 <!--
-    Optional
-    In case of breaking changes, functionality changes or refactors, please add release notes here. 
-    For more information, please see [the contributing guidelines](https://os.mbed.com/docs/mbed-os/latest/contributing/workflow.html#pull-request-types).
+    This section is compulsory for all Feature and Major PR types.
```

Comment:
For the features and major PRs we need only "summary of changes", for breaking changes we need all three subsections.

Reply:
Major update is a breaking change?

Reply:
OK updated the comments accordingly